.gitignore

```diff
@@ -7,6 +7,9 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore RubyMine dependencies
+/.idea
```

Comment:
I usually ignore .idea using **.idea/**

@petrucioata @biancabordianu @AnamariaaNistor  is this working? What is the exact difference between using the **/** character at the beginning or the end?

I thought that if it's **something/** it will take everything under **something** and when using **/something** it takes the relative path to **something**

please let me know if I am mistaking